uni-app/src/web_app.rs

```diff
@@ -72,6 +73,9 @@ impl App {
             .unwrap()
             .unwrap()
             .append_child(&canvas);
+        js!{
+            @{&canvas}.focus();

```

Comment:
Would you mind to move [this line](https://github.com/jice-nospam/unrust/blob/81727b21c745dd68a5c5e2188af761021b2e9b8a/uni-app/src/web_app.rs#L63) to here too ? It seem to be doing the same thing but i didn't confirm that.

Reply:
In fact, for a canvas to be focusable, you need to add a tabindex to it. The call to focus() can only be done once the canvas is attached to the DOM, while the tabIndex=1 can be set at any time.
That's why the focus call is done after the append_child :
```
document()
            .query_selector("body")
            .unwrap()
            .unwrap()
.append_child(&canvas);
```
While the tabIndex property can be set along with other canvas properties (width, height).

Reply:
I see.  Everything's good for me. i would merge it.